app/atomic_gui.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from aiohttp_jinja2 import template
+
+from app.service.auth_svc import for_all_public_methods, check_authorization
+from app.utility.base_world import BaseWorld
+
+
+@for_all_public_methods(check_authorization)
+class AtomicGUI(BaseWorld):
+
+    def __init__(self, services, name, description):
+        self.auth_svc = services.get('auth_svc')
+        self.data_svc = services.get('data_svc')
+        self.name = name
+        self.description = description
+
+        self.log = logging.getLogger('atomic_gui')
+
+    @template('atomic.html')
+    async def splash(self, request):
+        abilities = [a for a in await self.data_svc.locate('abilities') if await a.which_plugin() == 'atomic']
+        adversaries = [a for a in await self.data_svc.locate('adversaries') if await a.which_plugin() == 'atomic']
```

Comment:
Since we don't actually generate any adversaries for atomic, can we just get rid of this? It's probably slowing down the plugin load since `which_plugin()` is walking the filesystem across all plugins for each loaded adversary and none will match `atomic`.   We can also remove it from the UI since it's just gonna show `0` for adversaries.

For future work, we should really do something like tag the abilities with metadata about what plugin loaded them at ingest time, or if we wanna stay the "walk the filesystem" route, create another method that is more like `is_loaded_by_plugin(plugin_name)` can can scan the plugin's data directory rather than every possible plugin.

Reply:
FWIW i did a quick test by skipping the adversary search  and it didn't really impact performance in any meaningful way on my machine since there were only like 29 adversaries loaded in total outside of atomic (bummer). Still probably ok to remove it since it's just gonna always show 0 for atomic.